how-to/divio-app.rst

```diff
@@ -0,0 +1,126 @@
+.. _divio-app:
+
+How to install and use the Divio app
+====================================
+
+The Divio app helps you run your Divio projects locally in our Docker environment, upload and download media, database
+and configuration changes as well as set up a shell environment ready for interaction with the project running in the
+Docker container.
+
+
+Set up the Divio app
+--------------------
+
+
+Installation
+~~~~~~~~~~~~
+
+* **Download** the `Divio app here <https://divio.com/developers/#app>`_, which is available for Macintosh, Linux and
+  Windows.
+
+* Open the downloaded package and follow the on-screen instructions to install the application.
+
+.. _note:: 
+
+  Some anti-virus software for Windows may warn that the Divio app application is unsigned and therefore represents a
+  security risk. You can safely ignore this warning.
+
+
+Loging in
+~~~~~~~~~
+
+Once the application is installed and running you can login using your email address and password, or if you singed up
+with Google or GitHub, simply copy `the access token <https://control.divio.com/account/desktop-app/access-token/>`_
+from the Divio control panel and paste it into the provided field.
+
+Select a workspace folder on your computer where you’d keep all the Divio cloud project files.
+
+
+Use the Divio app
+-----------------
+
+Select a project to work with from the list. By default, the list shows all projects, but you can narrow down the
+selection by choosing a particular organisation from the dropdown menu.
+
+
+Set up your local project
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+* Select a project from the list.
+* Hit *set up project* to get started.
+
+The Divio app will run through a number of processes to set up the local project. It all happens automatically and takes
+just a few minutes.
+
+
+Launch the site locally
+~~~~~~~~~~~~~~~~~~~~~~~
+
+Once the set up process has completed, you can start the local site by launching the local server. 
+
+* Hit **Start** to launch the project.
+* If the project has been deployed, you can now open the local site in your browser by clicking on the *eye icon*.
+
+
+Edit project files
+~~~~~~~~~~~~~~~~~~
+Hit the *folder icon* to open the project folder, use your favorite editor to edit the project files.
+
+
+Upload/download changes
+~~~~~~~~~~~~~~~~~~~~~~~
+
+Use the *Download code* / *Upload code* buttons to push / pull your changes to / from the cloud.
+
+You can also download / upload media files or database changes using the downarorw icon.
```

Comment:
typo here, but anyway i'd say "using the dropdown"

Reply:
changed it to dropdown